types/office-js-preview/index.d.ts

```diff
@@ -13098,6 +13207,48 @@ declare namespace Office {
          * @param userContext - Optional. Any state data that is passed to the asynchronous method.
          */
         getUserIdentityTokenAsync(callback: (asyncResult: Office.AsyncResult<string>) => void, userContext?: any): void;
+        /**
+         * Gets if the client signature is enabled.
+         * 
+         * For Windows and Mac rich clients, if the signature is set to a template in a new email, forward, or reply
```

Comment:
For Windows and Mac rich clients, if the default signature for New Messages or Replies/Forwards is set to a template for the sending Outlook account, the API call should return `true`.

Reply:
Updated